Week7/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/data/networking/RemoteApi.kt

```diff
@@ -0,0 +1,21 @@
+package com.raywenderlich.myfavoritemovies.data.networking
+
+import com.raywenderlich.myfavoritemovies.App
+import com.raywenderlich.myfavoritemovies.model.Failure
+import com.raywenderlich.myfavoritemovies.model.Result
+import com.raywenderlich.myfavoritemovies.model.Success
+import com.raywenderlich.myfavoritemovies.model.response.MovieTopLevelResponse
+
+
+const val BASE_URL = "https://www.myapifilms.com"
+
+class RemoteApi(private val apiService: RemoteApiService) {
```

Comment:
I'm note sure why you have this class, to fetch only single movies, why you have Repository classes, to fetch all movies, and then why you kind of avoid using the repository, and you use this to directly fetch data.